linode/nb/resource_test.go

```diff
@@ -113,6 +113,44 @@ func TestAccResourceNodeBalancer_update(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: tmpl.Basic(t, nodebalancerName, testRegion),
```

Comment:
Should we use  `tmpl.Firewall` to config this test case and use `tmpl.Basic` for the `update` testing instead? We can also check if the `firewall_id` has been set in this test case.